src/main/java/com/revature/services/TripService.java

```diff
@@ -1,15 +1,25 @@
 package com.revature.services;
 
 import com.revature.models.Trip;
+import com.revature.models.TripDTO;
 
 import java.util.List;
 import java.util.Optional;
 
 public interface TripService {
```

Comment:
This seems to have been left over from previously, but please change the spaces on the older methods to tabs.

Reply:
I don't see any spaces here, do you mean the double line returns? Pretty sure that is standard for changes of publisher in imports.

Reply:
The lines below actually. Inside the interface. I commented too high.